src/routes/api.rs

```diff
@@ -79,7 +80,13 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimit) {
           .route("/list", web::get().to(route_get::<GetPosts>))
           .route("/like", web::post().to(route_post::<CreatePostLike>))
           .route("/save", web::put().to(route_post::<SavePost>))
-          .route("/join", web::post().to(route_post::<PostJoin>)),
+          .route("/join", web::post().to(route_post::<PostJoin>))
+          .route("/report", web::post().to(route_post::<CreatePostReport>))
+          .route(
+            "/report/resolve",
+            web::put().to(route_post::<ResolvePostReport>),
+          )
+          .route("/report/list", web::get().to(route_get::<ListPostReports>)),
```

Comment:
Forgot one more thing: these need to be documented in : https://dev.lemmy.ml/docs/contributing_websocket_http_api.html / `docs/src/contributing_websocke_http_api.md`

Reply:
added, unsure of how to regenerate the TOC

Reply:
Thx. I can do that. In the future its:

```
npm i -g markdown-toc
markdown-toc -i contributing...md
```

Reply:
Please update the TOC for apub outline as well, I forgot that last time I changed it.